src/components/transcript/index.tsx

```diff
@@ -109,12 +110,13 @@ const Transcript = ({ reviewData }: { reviewData: UserReview }) => {
 
   const [submitState, setSubmitState] =
     useState<SubmitState>(defaultSubmitState);
-
+  const editorRef = useRef<MdEditor | null>(null);
   const toast = useToast();
   const { isOpen, onOpen, onClose } = useDisclosure();
 
   const restoreOriginal = () => {
     if (!transcriptData?.originalContent) return;
+    editorRef.current?.setText(transcriptData.originalContent?.body);
```

Comment:
I'm thinking of a fallback `transcriptData.originalContent?.body ?? ""` but that was due to a legacy problem.
Undefined crashed the previous editor, I do not know of this. @0tuedon check if undefined crashes the editor. They may already have an internal fallback

Reply:
okay

Reply:
i don't think it does, but let me add just incase

Reply:
@Emmanuel-Develops  i just tested and undefined doesn't crash it,it just shows empty string
![Uploading Screenshot 2023-07-11 at 13.58.05.png…]()

Reply:
If no issues then you can leave it as is. Is the text editor responsive and changes to that reflect in the state?

Reply:
Alright then 👍